src/bindings/cpp/benchmarks/benchmark_thread.cpp

```diff
@@ -67,8 +67,9 @@ class Environment : public ThreadBoolean
 	: ThreadBoolean (ks, context_, Key ("/test/%layer1%", KEY_CASCADING_NAME, KEY_META, "default", "1", KEY_END)),
 	  nested (ks, context_), person (ks, context_),
 	  profile (ks, context_, Key ("/%layer1%/profile", KEY_CASCADING_NAME, KEY_META, "default", "default", KEY_END)),
-	  bm (ks, context_, Key ("/%layer1%/%layer2%/%layer3%/%layer4%/%layer5%/%layer6%/%layer7%/%layer8%/%layer9%/", KEY_CASCADING_NAME,
```

Comment:
> Did you use clang-format-3.8? Maybe @sanssecours can give you a tip how to get it (on Mac).

You can install `clang-3.8` via `brew install clang-format@3.8`.

> If we would use different versions of `clang-format` we would keep reverting our changes.

We could also just use the current version (`5.0.0`) of the software.